src/analyzer_main.cpp

```diff
@@ -559,6 +560,58 @@ struct Analyzer {
             return false;
         }
 
+        //Handling susceptibility
+        if(config.use_susceptibility == true && stats.n_steps % 10 == 0){
+
+            ofstream myfile;
+
+            stringstream ss;
+            ss << "output/change_in_ideology_step_" << stats.n_steps << ".dat";
+            string filename = ss.str();
+            myfile.open (filename.c_str());
+
+            myfile << "\nn_steps: " << stats.n_steps;
+
+            int old_ideology;
+
+            for (Agent& agent : network) {
+                if (agent.susceptibility == 1.0) {
+
+                    myfile << "\n\nagent_id: " << agent.id;
+
+                    old_ideology = agent.ideology_bin;                
+
+                    // Looking at the ideology of all of an agent's followings:
+                    int counts[N_BIN_IDEOLOGIES];
+                    for (int& count : counts) {
+                        count = 0; 
+                        // Counts start at 0
```

Comment:
Nit: Comments before the code or on the same line. Eg `count = 0; // Counts start at 0`, but not on the next line.